doc/Developing.md

```diff
@@ -50,7 +50,16 @@ tab with the frontend.
 
 ## Run the electron-based example application
 
-It can also be started with:
+From the root directory run:
+
+    npm run rebuild:electron
+
+This command rebuilds native node packages against the version of node used by electron.
+To rollback changes run:
+
+    npm run rebuild:web
+
+It can also be started with from examples/electron directory:
```

Comment:
It could be that we can have a setup there we exclude node-pty from hoisting and don't need to run rebuild, but I've spent half a day and failed to come up with such. I think mostly we work with one or another example anyway and later with more experience maybe can overcome it.